pkgs/tools/misc/claws/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "claws";
+  version = "0.3.2";
+
+  goPackagePath = "github.com/thehowl/claws";
```

Comment:
could use `"github.com/thehowl/${pname}"; though this is probably personal preference